src/sentry/scim/endpoints/teams.py

```diff
@@ -201,7 +202,34 @@ def _get_team(self, organization, team_id):
             raise Team.DoesNotExist
         return team
 
+    @extend_schema(
+        operation_id="Query an Individual Team",
+        parameters=[SCIM_PARAMS.TEAM_ID, GLOBAL_PARAMS.ORG_SLUG],
+        request=None,
+        responses={
+            200: TeamSCIMSerializer,
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOTFOUND,
+        },
+        examples=[  # TODO: see if this can go on serializer object instead
```

Comment:
Can it?

Reply:
in the docs there is an example of adding an example to a serializer: https://drf-spectacular.readthedocs.io/en/latest/customization.html#step-4-extend-schema-serializer

I haven't spent the time to look into how this gets picked up as an example on endpoints, but it could potentially be useful.